pkg/component/controller/kuberouter.go

```diff
@@ -313,6 +315,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: POD_NAME
```

Comment:
This (and the RBAC change) has been introduced in [`afdf553f add loadbalancer address allocator`](https://github.com/cloudnativelabs/kube-router/commit/afdf553fa8b3eae68fcd48f4ed1d89ac31eb3310).

> This adds a simple controller that will watch for services of type LoadBalancer and try to allocated addresses from the specified IPv4 and/or IPv6 ranges. It's assumed that kube-router (or another network controller) will announce the addresses.
> 
> As the controller uses leases for leader election and updates the service status new RBAC permissions are required.

Is that something that we need to be aware of?

Reply:
Partially yes. I mean we do provide `extraArgs` via which user can enable LB feature. Thus IMO we should have these in place.